src/main/java/com/CezaryZal/manager/db/service/UserService.java

```diff
@@ -2,15 +2,16 @@
 
 import com.CezaryZal.entity.User;
 
-import java.util.Optional;
+import javax.naming.NameNotFoundException;
+import java.util.List;
 
 public interface UserService {
 
-    Optional<User> findById(Long index);
+    User findById(Long index) throws NameNotFoundException;
```

Comment:
szukasz usera po ID jak nie znajdziesz to rzucasz exception że namenotfound - niespójne troche
może po prostu powinien być exception usernotfound

Reply:
OK